content/spin/v2/manifest-reference.md

```diff
@@ -129,7 +129,7 @@ Each array entry contains a mix of common fields and trigger-specific fields.
 
 ## The `component` Table
 
-The keys of the `component` table, usually written as part of the table syntax e.g. `[component.my-component]`, are user-defined.  (In the preceding example, the key is `my-component`). Component names must be kebab-cased, i.e. the only permitted separator is a hyphen.
+The keys of the `component` table, usually written using TOML table syntax e.g. `[component.my-component]`, are ids of components.  Component ids must be kebab-cased, i.e. the only permitted separator is a hyphen.  In the following example, `my-component` is used as an example component id.
```

Comment:
We should probably come to an agreement on IDs vs names and apply it throughout.  I know I have gone back and forth.  In v1 I think it was mostly 'ID', but I believe v2 favoured 'name' because of alignment with component model terminology?  I am not sure.

(Also 'name' finesses needing to agree a standard on 'id' vs 'ID' _grin_)